### PR TITLE
Removed all deprecated ScriptCanvas library files

### PR DESCRIPTION
**Pull request by lsemp3d**
## What does this PR do?

Many Script Canvas files were deprecated and superseded by a new auto-registration method. The code removed is no longer used and contributes to build times, removing it to reduce unnecessary compilations.

### CODE REVIEW

**Review by cgalvan — APPROVED** [review]

**Review by dtamkin1 — APPROVED** [review]

File: Gems/ScriptCanvasTesting/Assets/ScriptCanvas/Tests/$tmp17421_Test_LY_79396.scriptcanvas
Comment (lawsonamzn) [inline Gems/ScriptCanvasTesting/Assets/ScriptCanvas/Tests/$tmp17421_Test_LY_79396.scriptcanvas:1]: id you mean to check this in?

**Review by lawsonamzn — APPROVED** [review]

**Comment by AMZN-alexpete** [comment]
Nice to see redcode :) If the removed code/APIs were accessible, can you please update the `gem.json` `version` to `1.0.0` to indicate a breaking change was made in case anyone was using that code?  More details here:  https://github.com/o3de/community/blob/main/guides/o3de-deprecation-guidelines.md#api-deprecation---final-steps

File: Gems/ScriptCanvasTesting/Assets/ScriptCanvas/Tests/$tmp17421_Test_LY_79396.scriptcanvas
Comment (lsemp3d) [reply Gems/ScriptCanvasTesting/Assets/ScriptCanvas/Tests/$tmp17421_Test_LY_79396.scriptcanvas:1]: No, will remove

**Review by lawsonamzn — APPROVED** [review]

**Comment by amzn-changml** [comment]
The build is green but the status is not being updated. Submitting an override for the PR merge status.

**Pull request merged**